docker/WORKSPACE.docker

```diff
@@ -31,11 +31,17 @@ git_repository(
     shallow_since = "1570114335 -0400",
 )
 
+# External dependency for torch_tensorrt if you already have precompiled binaries.
```

Comment:
Can we potentially leverage another workspace file we have like the CI ones so that we dont need to update as many files every time we update the python version?

Reply:
Alternatively we could make it all new_local_repositories and rely on the dockerfile to install from requirements.txt